server/app/controllers/fusor/api/v2/deployments_controller.rb

```diff
@@ -47,5 +47,10 @@ def find_deployment
       not_found and return false if params[:id].blank?
       @deployment = Deployment.find(params[:id])
     end
+
+    def deploy
```

Comment:
We can probably remove this action.  A deploy action was added with my PR, which was merged earlier today.  It is on approx line 41 of the controller.